src/detail-panels/account-header/handle-history/handle-history.jsx

```diff
@@ -60,79 +65,101 @@ function ChangeEntry({ date, handle, pds, lastHandle, lastPds, isCreated }) {
   const isMigratedToMushroom =
     isPdsChanged &&
     pds.endsWith('bsky.network') &&
-    lastPds && lastPds.endsWith('bsky.social');
-  const isMigratedExternally =
-    isPdsChanged &&
-    !pds.endsWith('bsky.network');
-  
-  if (isCreated) return (
-    <div className='event-entry'>
-      {date ? <FormatTimestamp className='event-timestamp' timestamp={date} /> : 'unknown date '}
-      <span className='event-label event-label-registered'>
-        {localise('registered ', { uk: 'зареєстровано ' })}
-      </span>
-      {
-        !handle ? undefined :
+    lastPds &&
+    lastPds.endsWith('bsky.social');
+  const isMigratedExternally = isPdsChanged && !pds.endsWith('bsky.network');
+
+  // getting elapsed time
+  function getRelativeTime(dateString) {
+    const newDate = new Date(dateString);
+    const now = new Date();
+    const timeDiffInMillis = newDate.getTime() - now.getTime();
+    const daysDiff = Math.round(timeDiffInMillis / (1000 * 60 * 60 * 24));
+
+    const inferred = new Intl.RelativeTimeFormat(undefined, {
+      numeric: 'auto',
+    });
+    return inferred.format(daysDiff, 'days');
+  }
+
+  if (isCreated)
+    return (
+      <div className="event-entry">
+        {!handle ? undefined : (
           <>
-            {localise('as ', { uk: 'як ' })}
+            {localise('User ', { uk: 'як ' })}
```

Comment:
when completely changing a string like this the old Ukrainian translation no longer applies and should be removed. (other changes are subtle enough to not matter imo)

Reply:
Oh yeah this makes sense thank you I can change it